etc/cumulus-config/cumulus-config.yml

```diff
@@ -82,6 +82,7 @@ cumulus_unlimited_quotas:
   ram: -1
   secgroup_rules: -1
   secgroups: -1
+  volumes: -1
```

Comment:
We should probably stop doing unlimited, but this gets us to a consistent place, so +1